test_wazuh/test_fim/test_windows_registry/data/wazuh_conf.yaml

```
@@ -0,0 +1,44 @@
+---
+# conf 1
+- tags:
+  - ossec_conf
+  apply_to_modules:
+  - test_windows_registry
+  section: syscheck
+  elements:
+  - disabled:
+      value: 'no'
+  - frequency:
+      value: '2'
+  - windows_registry:
+      value: 'HKEY_LOCAL_MACHINE\SOFTWARE\\Classes\\testkey'
```

```suggestion
      value: WINDOWS_REGISTRY
```
Use wildcard since it is the same value.